tsconfig.json

```diff
@@ -12,7 +12,7 @@
       "DOM.iterable"
     ],
     "module": "es2020",
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
https://github.com/rollup/rollup/issues/5199#issuecomment-1782231154 🤷

Reply:
we use `bundler` in all of our oas tooling as well fwiw

Reply:
that makes me feel a lot better about this, thank you!